### ISSUE TITLE
Array to string conversion on PHP 5.6.x

### ISSUE THREAD

**Issue by redboxmarcins** [shipperhq/module-shipper#6]
Logger helper is causing error that array is converted to string

example:

`$this->logger->debug(array($module, $message,$data),$context);`

**Comment by wsagen** [comment]
Hi,

Thanks for raising. Sounds like you aren't running shipperhq/module-logger. This is now a dependency for ShipperHQ, it should install automatically when using composer installation. If you can update via composer it should install the logger and error should be resolved. 

Please let me know if you still see the error.

**Comment by wsagen** [comment]
Closing as cannot reproduce

**Comment by cheelahim** [comment]
Hey @wsagen! Actually this is exactly from the shipperhq/module-logger package. Check ShipperHQ\Shipper\Helper\LogAssist.

```
    public function postDebug($module, $message, $data, array $context = array()) {
        if ($this->shipperDataHelper->getConfigValue('carriers/shipper/debug')) {
            $this->logger->debug(array($module, $message,$data),$context);
        }
    }
```
This does comply with Psr/LoggerInterface. String value is expected as a first (message) argument while array is passed.
Could you please fix it and release along with #7

**Comment by cheelahim** [comment]
Hey @wsagen! The shipperhq/module-logger is installed. But check ShipperHQ\Shipper\Helper\LogAssist.

```
   public function postDebug($module, $message, $data, array $context = array()) {
        if ($this->shipperDataHelper->getConfigValue('carriers/shipper/debug')) {
            $this->logger->debug(array($module, $message,$data),$context);
        }
    }
```
The way it's invoked does not comply with Psr/LoggerInterface. String value is expected as a first (message) argument while array is passed.
Could you please fix it and release along with #7

**Comment by wsagen** [comment]
This is now resolved in release version 20.0.6
